scripts/modules/b2b-account/b2b-account-request.js

```diff
@@ -46,7 +47,7 @@ define(["modules/jquery-mozu", "hyprlive", "modules/models-b2b-account", 'module
             "userName": $("#userEmail").val().trim(),
             "firstName": $("#userFirstName").val().trim(),
             "lastName": $("#userLastName").val().trim(),
-            "localeCode": "en-US",
+            "localeCode": apiContext.headers['x-vol-locale'],
```

Comment:
fallback to `en-US` locale, in case `apiContext` is null or `x-vol-locale` is not present in the headers.
Do same thing for all the other changes.

Reply:
Made the changes